keras_retinanet/models/densenet.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright 2018 vidosits (https://github.com/vidosits/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import keras
+from keras.applications.densenet import DenseNet, get_file
+
+from ..models import retinanet
+
+BASE_WEIGHT_PATH = 'https://github.com/fchollet/deep-learning-models/releases/download/v0.8/'
```

Comment:
I prefer this to be :

```python
WEIGHT_PATH = 'https://github.com/fchollet/deep-learning-models/releases/download/v0.8/densenet{}_weights_tf_dim_ordering_tf_kernels_notop.h5'
```

So that later on we can do:

```python
weights_url = WEIGHT_PATH.format(backbone)
```